app/views/home/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<% flash.each do |type, msg| %>
+        <div>
+          <p class="notice"><%= notice %></p>
+          <p class="alert"><%= alert %></p>
+        </div>
+  <% end %>
+<main class="d-flex flex-column align-items-center">
+
+  <% if current_user %>
+    <h2 class><%= "Welcome #{current_user.name} " %></h2>
+    <script>
+      window.location.href = '/groups';
+    </script>
+  <% else %>
+    <a href="/users/sign_in">
+      <button type="button" class="btn btn-primary m-2 mt-5">Log in</button>
+    </a>
+    <a href="/users/sign_up">
+      <button type="button" class="btn btn-secondary m-2">Sign Up</button>
+    </a>
```

Comment:
- [ ] `Required`
- [ ] Amazing job so far 👏 
- [ ] However, please the splash screen as the template, in accordance with the requirements.
- [ ] and a logo or your apps name
- [ ] please see screenshots below 🥂 

Original template
![image](https://github.com/gealsanchez/budget_app/assets/66230306/1d816c0b-baca-425e-b523-61680aaae8c3)

Your template
![image](https://github.com/gealsanchez/budget_app/assets/66230306/91d7a377-a669-4dd7-bd3c-d62f51645900)